tests/test_cli.py

```diff
@@ -98,6 +102,37 @@ def test_notebook_launcher(self):
         with patch_environment(omp_num_threads=1, accelerate_num_processes=2):
             run_command(cmd, env=os.environ.copy())
 
+    def test_mpi_multicpu_config_cmd(self):
+        """
+        Parses a launch command with a test file and the 0_28_0_mpi.yaml config. Tests getting the command and
+        environment vars and verifies the mpirun command arg values.
+        """
+        mpi_config_path = str(self.test_config_path / "0_28_0_mpi.yaml")
+        test_file_arg = "--cpu"
+        sys.argv = ["accelerate", str(self.test_file_path), test_file_arg]
```

Comment:
Not sure if it is a good idea to overwrite `sys.argv`. Could this be mocked instead? My concern is that it could have side effects on other tests, as it seems that this is not cleaned up automatically:

```python
class TestFoo:
    def test_foo(self):
        assert sys.argv != ["foobar"]  # passes

    def test_bar(self):
        sys.argv = ["foobar"]
        assert True

    def test_baz(self):
        assert sys.argv != ["foobar"]  # fails
```

Also not sure if this could not have some weird interactions with the test runner, since the runner also reads from `sys.argv`. Probably only once when it launches, but who knows, it would be hard to debug if this caused an error.